client/src/CardComponent.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./CardComponent.css";
+import Tag from "./Tag.js";
+
+/*Large container: holds two parts*/
+/*Top part: text + image area*/
+/*Top part: text + image area*/
+/*Bottom part: flexbox left justified container*/
+/*Flexbox left justified containers: holds a map of tag parts*/
+
+/* calcuate the width of the page and make things look good accordingly*/
+
+function CardComponent(props) {
+  return (
+    <div>
+      <div className="cardContainer">
+        <div className="pic-text-container">
+          <img className="profile-pic" alt="Image" src={props.profile} />
+          <div className="nameContainer">
+            <b id="CoOp-Name">{props.name}</b>
+            <div id="CoOp-Location">{props.location}</div>
+          </div>
+        </div>
+        <div className="tagsContainer">
+          {props.tags.map((text) => (
```

Comment:
Oh also, let's add a null check for `props.tags` so that it doesn't error in the case that there aren't any `tag`s.